go/mysql/binlog_event_compression.go

```diff
@@ -41,35 +44,63 @@ const (
 	payloadUncompressedSizeField
 )
 
-// Compression algorithms that are supported (only zstd today
-// in MySQL 8.0):
-// https://dev.mysql.com/doc/refman/8.0/en/binary-log-transaction-compression.html
 const (
+	// Compression algorithms that are supported (only zstd today
+	// in MySQL 8.0):
+	// https://dev.mysql.com/doc/refman/8.0/en/binary-log-transaction-compression.html
 	TransactionPayloadCompressionZstd = 0
 	TransactionPayloadCompressionNone = 255
-)
 
-var TransactionPayloadCompressionTypes = map[uint64]string{
-	TransactionPayloadCompressionZstd: "ZSTD",
-	TransactionPayloadCompressionNone: "NONE",
-}
+	// Length of the binlog event header in the transaction payload.
+	headerLen = BinlogEventLenOffset + 4
```

Comment:
Does `4` stand for `BinlogEventTypeOffset` here? Otherwise please comment what it stands for.

Reply:
It's the uint32 where the event length is stored. I'll add a const for that.

Reply:
Improved things here: https://github.com/vitessio/vitess/pull/16328/commits/d5cbf05d3b0269a36a225b8b13b1952038c7f689

Thanks!